azure-pipelines.yml

```diff
@@ -17,14 +17,40 @@ trigger:
 jobs:
 
 - job: CPU_Benchmarks
-
   timeoutInMinutes: 0
 
   pool:
     name: Default
     demands: 
-      - NO_GPU
       - Agent.OS -equals Linux
+
+  strategy:
+    matrix:
+      GRIDREC:
+        algorithmName: 'gridrec'
+      ART:
+        algorithmName: 'art'
+      BART:
+        algorithmName: 'bart'
+      MLEM:
+        algorithmName: 'mlem'
+      OSEM: 
+        algorithmName: 'osem'
+      OSPML_Hybrid:
+        algorithmName: 'ospml_hybrid'
+      OSPML_Quad: 
+        algorithmName: 'ospml_quad'
+      PML_Hybrid: 
+        algorithmName: 'pml_hybrid'
+      PML_Quad: 
+        algorithmName: 'pml_quad'
+      SIRT:
+        algorithmName: 'sirt'
+      SIRT_CPU:
+        algorithmName: 'sirt_cpu'
```

Comment:
Be consistent with naming scheme. 'sirt' should always refer to CPU algorithm. 'sirt_gpu' always refer to GPU algorithm. There should be no 'sirt_cpu'.